QRCoder/QRCodeGenerator/AlphanumericEncoder.cs

```diff
@@ -7,38 +7,34 @@ public partial class QRCodeGenerator
     /// </summary>
     private static class AlphanumericEncoder
     {
-        private static readonly char[] _alphanumEncTable = { ' ', '$', '%', '*', '+', '-', '.', '/', ':' };
-
-        /// <summary>
-        /// A dictionary mapping alphanumeric characters to their respective positions used in QR code encoding.
-        /// This includes digits 0-9, uppercase letters A-Z, and some special characters.
-        /// </summary>
-        private static readonly Dictionary<char, int> _alphanumEncDict = CreateAlphanumEncDict(_alphanumEncTable);
-
-        /// <summary>
-        /// Creates a dictionary mapping alphanumeric characters to their respective positions used in QR code encoding.
-        /// This includes digits 0-9, uppercase letters A-Z, and some special characters.
-        /// </summary>
-        /// <returns>A dictionary mapping each supported alphanumeric character to its corresponding value.</returns>
-        private static Dictionary<char, int> CreateAlphanumEncDict(char[] alphanumEncTable)
-        {
-            var localAlphanumEncDict = new Dictionary<char, int>(45);
-            // Add 0-9
-            for (char c = '0'; c <= '9'; c++)
-                localAlphanumEncDict.Add(c, c - '0');
-            // Add uppercase alphabetic characters.
-            for (char c = 'A'; c <= 'Z'; c++)
-                localAlphanumEncDict.Add(c, localAlphanumEncDict.Count);
-            // Add special characters from a predefined table.
-            for (int i = 0; i < _alphanumEncTable.Length; i++)
-                localAlphanumEncDict.Add(alphanumEncTable[i], localAlphanumEncDict.Count);
-            return localAlphanumEncDict;
-        }
+#if HAS_SPAN
+        // With C# 7.3 and later, this byte array is inlined into the assembly's read-only data section, improving performance and reducing memory usage.
+        // See: https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-core-3-0/
```

Comment:
```suggestion
        // See: https://vcsjones.dev/csharp-readonly-span-bytes-static
```
This link is more concise.

Reply:
True, but I'd prefer to link to official MS docs.